source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,210 @@
+.. _csharp-delete-guide:
+
+=================
+Delete a Document
```

Comment:
issue: The title doesn't exactly match the page content. I expected it to only contain information about how to delete *a* document. Perhaps simply "Delete Documents"?

Reply:
Title was taken from the corresponding GoDoc page, but I agree, the title should be more accurate even if it means changing the task oriented language